cmd/apiserver/main.go

```diff
@@ -129,3 +135,7 @@ func apiKeyFunc() httplimit.KeyFunc {
 		return ipKeyFunc(r)
 	}
 }
+
+func handleChaff(tracker *chaff.Tracker, next http.Handler) http.Handler {
```

Comment:
I'm not a huge fan of one-line helpers, but the handler becomes really difficult to follow without it.